2996_reflection/reflection.md

```diff
@@ -5630,6 +5630,7 @@ consteval bool is_data_member_spec(info r);
 - [#.#]{.pnum} `$C$` is incomplete from every point in the evaluation context,
 - [#.#]{.pnum} `is_data_member_spec(@$r$~$K$~@)` is `true` for every `@$r$~$K$~@` in `mdescrs`, and
```

Comment:
move the "and" down to the next bullet, which has to end with ", and" instead of "."